src/pages/networks/NetworkPeers.tsx

```diff
@@ -0,0 +1,149 @@
+import type { FC } from "react";
+import {
+  EmptyState,
+  Icon,
+  MainTable,
+  Row,
+  ScrollableTable,
+  useNotify,
+  Spinner,
+} from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import type { LxdNetwork } from "types/network";
+import { useDocs } from "context/useDocs";
+import { fetchNetworkPeers } from "api/network-peering";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  network: LxdNetwork;
+  project: string;
+}
+
+const NetworkPeers: FC<Props> = ({ network, project }) => {
+  const docBaseLink = useDocs();
+  const notify = useNotify();
+
+  const {
+    data: peers = [],
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: [
+      queryKeys.projects,
+      project,
+      queryKeys.networks,
+      network.name,
+      queryKeys.peers,
+    ],
+    queryFn: async () => fetchNetworkPeers(network.name, project),
+  });
+
+  if (error) {
+    notify.failure("Loading local network peers failed", error);
```

Comment:
```suggestion
    notify.failure("Loading local network peerings failed", error);
```
Let's stay consistent with the naming from the menu / heading.